src/components/Card/CardStats.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement } from "react";
+import styled from "styled-components";
+import { Heading1, Text } from "../../typography";
+
+export interface CardStatsDetails {
+  label: string;
+  value: string;
+}
+
+export interface CardStatsProps {
+  details: CardStatsDetails[];
+}
+
+export const CardStats = ({ ...props }: CardStatsProps): ReactElement => (
```

Comment:
We can do one or the other, doesn't really make sense to spread props here:

```suggestion
export const CardStats = (props: CardStatsProps): ReactElement => (
```

or

```suggestion
export const CardStats = ({ details, ... }: CardStatsProps): ReactElement => (
```

Reply:
Fixed with f59d71e !